usaspending_api/etl/management/commands/elasticsearch_indexer.py

```diff
@@ -180,7 +186,7 @@ def parse_cli_args(options: dict, es_client) -> dict:
             logger.error(f"Write alias '{config['write_alias']}' is missing")
             raise SystemExit(1)
     else:
-        if es_client.indices.exists(config["index_name"]):
+        if config["index_name"] and es_client.indices.exists(config["index_name"]):
             logger.error(f"Data load into existing index. Change index name or run an incremental load")
             raise SystemExit(1)
```

Comment:
Comment is for code LN 181 - 191. Don't the delete operations require the write alias to be present?

Self-investigative answer: No, because the delete code is very oddly using the `query_alias_prefix` config value. Odd.

Still, I think the delete-only workflow should still ensure the target index or alias exists rather than attempt to delete records from a non-existent index